### DISCUSSION TITLE
Search mkdocs feature

### DISCUSSION THREAD

**Discussion by vegeta123254** [mkdocs/mkdocs#2451] [Q&A]
I am using mkdocs-windmill theme, 
And I want to use the search feature to use in another application,
I mean the search to be embedded in another application.
Is that possible?

**Comment by waylan** [comment]
I'm not sure I understand. Do you want to implement a similar search feature in another application or do you want to search your MkDocs site form another application?

In any event, MkDocs uses [Lunr.js](http://lunrjs.com/) as its search engine. You are free to use that same library in your application. Whether you use it to generate yoru own search index or point as MkDocs' index is up to you.

**Reply by vegeta123254** [reply]
I want to search my mkdocs site from another application.
Could you please let me know how to use Lunr.js to use Mkdocs' index?
I was trying to use the search_index.json containing the location, text and title to replicate the response of search .
